syntax = "proto3";

message UserCreated {
  string user_id = 1;
  string email = 2;
  string created_at = 3;
}